src/main/java/org/assertj/core/api/AssertionsForClassTypes.java

```diff
@@ -85,6 +86,19 @@
   }
 
   /**
+   * Create assertion for {@link java.util.concurrent.CompletionStage}.
+   *
+   * @param actual the actual value.
+   * @param <RESULT> the type of the value contained in the {@link java.util.concurrent.CompletionStage}.
+   *
+   * @return the created assertion object.
+   */
+  @CheckReturnValue
+  public static <RESULT> CompletionStageAssert<RESULT> assertThat(CompletionStage<RESULT> actual) {
```

Comment:
move this method should be in `AssertionsForInterfaceTypes` as `CompletionStage` is an interface